module/observationSubmissions/helper.js

```diff
@@ -1108,7 +1108,7 @@ module.exports = class ObservationSubmissionsHelper {
           {
             _id: { $in: entityIds },
             tenantId: tenantData.tenantId,
-            orgId: {$in:['ALL',tenantData.orgId ]}
+            orgIds: {$in:['ALL',tenantData.orgId ]}
```

Comment:
@borkarsaish65 does this model also need change or is it orgIds now?

Reply:
this is entity management call , it was wrong I corrected it